lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -464,7 +464,7 @@ flt-glass-pane * {
       // VisualViewport API is not enabled in Firefox as well. On the other hand
       // Firefox returns correct values for innerHeight, innerWidth.
       // Firefox also triggers html.window.onResize therefore we don't need this
-      // timer setup for Firefox.
+      // timer to be set up for Firefox.
```

Comment:
I'd argue this was correct before, but I'll take it 🙂

Reply:
Haha. I sat there for about 30 seconds debating whether to leave it ambiguous as to whether it was a "timer setup" or needing to have the timer set up before giving up and deciding to eliminate the ambiguity with a "to be".